containers/src/Kokkos_DualView.hpp

```diff
@@ -953,10 +953,29 @@ class DualView : public ViewTraits<DataType, Arg1Type, Arg2Type, Arg3Type> {
   ///
   /// This method only copies the old contents into the new View
   /// objects for the device which was last marked as modified.
-  template <class... I>
+  template <class... ViewCtorArgs>
   void impl_resize(const size_t n0, const size_t n1, const size_t n2,
                    const size_t n3, const size_t n4, const size_t n5,
-                   const size_t n6, const size_t n7, const I&... arg_prop) {
+                   const size_t n6, const size_t n7,
+                   const Impl::ViewCtorProp<ViewCtorArgs...>& arg_prop) {
```

Comment:
Would it be too invasive of a change to make the param ordering of `impl_resize` to be consistent with `Kokkos::resize`?

Reply:
Not really, it was just the smallest change to the status quo.

Reply:
@ldh4 See f055b8c3e.